imap_processing/cdf/utils.py

```diff
@@ -72,6 +78,15 @@ def load_cdf(file_path: Path, **kwargs: dict) -> xr.Dataset:
         if isinstance(value, list) and len(value) == 1:
             dataset.attrs[attribute] = value[0]
 
+    # Remove attributes specific to xarray plotting from vars and coords
+    # TODO: This can be removed if/when feature is added to cdf_to_xarray to
+    #      make adding these attributes optional
+    if remove_xarray_attrs:
+        xarray_attrs = [v for k, v in ISTP_TO_XARRAY_ATTRS.items()]
+        for key in dataset.variables.keys():
+            for xarray_key in xarray_attrs:
```

Comment:
I think you're just keeping the values? (Similar in the other testing file where you iterate over items and then ignore the key, I'd just access `.values()` directly.
```suggestion
        for key in dataset.variables.keys():
            for xarray_key in  ISTP_TO_XARRAY_ATTRS.values():
```